lib/archethic/p2p/listener_protocol.ex

```diff
@@ -13,17 +17,22 @@ defmodule Archethic.P2P.ListenerProtocol do
   end
 
   def init({ref, transport, opts}) do
-    {:ok, socket} = :ranch.handshake(ref)
-    :ok = transport.setopts(socket, opts)
+    if :persistent_term.get(:archethic_up, nil) == nil do
```

Comment:
This could be refactored to reduce the two times `nil`usage.
Just by inverting the position:
```elixir
if :persistent_tem.get(:archethic_up, nil) do
    # Handle the connection
else
   {:stop, :node_bootstrapping}
end
```

This is just for clarity